### PR TITLE
Support window dragging with mouse-middle drag in xy charts

### LINKED ISSUE #732: Support window dragging with mouse-middle drag in xy charts
In comparison to Time Graph charts, the xy-chart hasn't implemented the window drag with `middle mouse button` drag. This shall be added to the XY-charts. The cursor shall be update to the hand icon during this operation.

#702 added support for this operation using CTRL+mouse left button drag.

### PR DESCRIPTION
**Pull request by hriday-panchasara**
Fixes #732

Signed-off-by: hriday-panchasara <hriday.panchasara@ericsson.com>

### CODE REVIEW

File: packages/react-components/src/components/xy-output-component.tsx [hunk 1]
```diff
@@ -241,6 +241,7 @@ export class XYOutputComponent extends AbstractXYOutputComponent<AbstractOutputP
     }
 
     private onWheel(wheel: React.WheelEvent): void {
+        console.log('onWheel called');
```
Comment (marco-miller) [inline packages/react-components/src/components/xy-output-component.tsx:244]: Should this be kept?

**Review by marco-miller — CHANGES_REQUESTED** [review]
Works well for me locally. Note, this PR fixes both `XY` and `TimeGraph` cases it seems. Without this patch, both cases are lacking mouse-middle button drag support, it seems. At least from my local point of view. And, I do see the hand pointer upon clicking that way.

File: packages/react-components/src/components/xy-output-component.tsx (hunk 1 above)
Comment (hriday-panchasara) [reply packages/react-components/src/components/xy-output-component.tsx:244]: Oops, forgot to take it out

**Review by marco-miller — APPROVED** [review]
LGTM, with that note I posted above / earlier. Thanks!

**Review by bhufmann — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**